nextjs/pages/components/Countdown.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+interface ICountdown {
```

Comment:
I react ville man normalt kalde den her for CountdownProps, men siden den ligger i en fil for sig selv havde jeg nok bare kaldt den props
```suggestion
interface Props {
```